zilencer/migrations/0065_remotepushdevice.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 5.2.3 on 2025-06-20 16:46
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0718_fix_topics_for_direct_messages"),
+        ("zilencer", "0001_squashed_0064_remotezulipserver_last_merge_base"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="RemotePushDevice",
+            fields=[
+                ("device_id", models.BigAutoField(primary_key=True, serialize=False)),
+                ("token", models.TextField()),
+                ("token_kind", models.PositiveSmallIntegerField(choices=[(1, "Apns"), (2, "Fcm")])),
```

Comment:
The capitalization here is pretty atypical. I think we should use standard spellings of these like we do in the `VisibilityPolicy` declaration.

Reply:
Changed it to:
```py
    class TokenKind(models.IntegerChoices):
        APNS = 1, "APNs"
        FCM = 2, "FCM"

    token_kind = models.PositiveSmallIntegerField(choices=TokenKind.choices)
```